modules/oauth-configuring-internal-oauth.adoc

```diff
@@ -3,21 +3,53 @@
 // * authentication/configuring-internal-oauth.adoc
 
 [id="oauth-configuring-internal-oauth_{context}"]
-= Configuring options for the internal OAuth server
+= Configuring the internal OAuth server's token duration
 
-You can configure default options for the internal OAuth server's 
+You can configure default options for the internal OAuth server's
 token duration.
 
+[IMPORTANT]
+====
+By default, tokens are only valid for 24 hours. Existing sessions
+expire after this time elapses.
+====
+
 .Procedure
 
-. Set the token duration options:
+. Create a configuration file that contains the token duration options:
 +
 [source,yaml]
 ----
-oauthConfig:
-  ...
+apiVersion: config.openshift.io/v1
+kind: OAuth
+metadata:
+  name: cluster
+spec:
   tokenConfig:
-    accessTokenMaxAgeSeconds: 86400 <1>
+    accessTokenMaxAgeSeconds: 172800 <1>
```

Comment:
The default lifetime is 86400, why there changed to 172800?

Reply:
I think you just want to give a example to change the default lifetime, it's better to have instruction here.

Reply:
I've updated the surrounding verbiage to be clearer that this procedure updates the default time to twice the default. Please take a look and let me know your thoughts.